oe_theme.theme

```diff
@@ -1721,6 +1721,30 @@ function oe_theme_theme_suggestions_oe_corporate_blocks_eu_footer_alter(array &$
  * Implements hook_preprocess_HOOK().
  */
 function oe_theme_preprocess_oe_corporate_blocks_ec_footer(array &$variables) {
+  $site_logo_href = 'https://commission.europa.eu';
+  // Ensure the site logo falls back to english version if the language is
+  // not EU language.
+  $language_code = \Drupal::languageManager()->getCurrentLanguage()->getId();
+  if (!EuropeanUnionLanguages::hasLanguage($language_code)) {
+    if ($language_code === 'nb') {
+      // Set correct icon for Norwegian language.
+      $variables['current_language_id'] = 'no';
+    }
```

Comment:
Do we have this somewhere else? This resembles as pt-pt to me. Can we make this logic reusable if that is the case?